README.md

```diff
@@ -45,6 +45,16 @@ $ docker service create \
   dockersamples/visualizer
 ```
 
+## Supported architectures
+
+The main `dockersamples/visualizer` image supports **linux/amd64**.
+
+**For ARM**, there is a pre-built image available. See [Running on ARM](#running-on-arm).
```

Comment:
We should be more specific about "arm" - this works for `armhf` only, it won't work for 64-bit ARM such as that found on Packet's or Scaleway's servers or the Pine64 etc.

Reply:
Makes sense. Done 👍